source/includes/azure/_public_ip_addresses.md

```diff
@@ -96,5 +99,47 @@ Attributes | &nbsp;
 `sku`  <br /> *string* | The sku of the public IP. Possible values: `BASIC`, `STANDARD`.
 `domainName` <br/>*string* | The subdomain part of the fqdn. This is only present if one is defined.
 `fqn` <br/>*string* | The fqdn which points to the public IP.
+`idleTimeout` <br/>*integer* | The number of minutes for the idleTimeout. I can be between 4 and 30 minutes.
 `allocationMethod` <br/>*string* | Allocation method of the public IP address. Possible values: `DYNAMIC`, `STATIC`.
 `ipVersion` <br/>*string* |  IP version of the public IP address. Possible values: `IPV4`, `IPV6`.
+
+
+<!-------------------- CREATE A PUBLIC IP -------------------->
+
+#### Create a public ip address
+
+```shell
+curl -X GET \
```

Comment:
should be POST

Reply:
my bad. Will be fixed soon.

Reply:
c75b48d4727301b98c91f510f6006c2c30b5e096